js/menu.js

```diff
@@ -1,18 +1,44 @@
-$('.show-menu-button').on('click', function() {
+/**
+ * showMenu() displays menu in smaller screens
```

Comment:
Already done, please check it here https://github.com/systers/FirstAide-web/pull/262/commits/aac881a07ce238abf294b5b87077278a091b9db9